backend/app/api/endpoints/admin/device_monitor.py

```
@@ -0,0 +1,287 @@
+        # Get online status
+        try:
+            provider_type = DeviceType(device_type_val)
+            provider = DeviceProviderFactory.get_provider(provider_type)
+            if provider:
+                device_status = await provider.get_status(
+                    db, device_kind.user_id, device_id
+                )
+                status_val = device_status.get("status", DeviceStatusEnum.OFFLINE.value)
+            else:
+                status_val = DeviceStatusEnum.OFFLINE.value
+        except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)